translations/fr.json

```diff
@@ -176,16 +171,16 @@
   "tribes.kickoo": "Kickoo",
   "tribes.kickoo.info": "Plages de sable blanc et cocotiers. Abondance de poissons et de fruits. Bienvenue chez les {0}. \n\nLes {0} commencent la partie avec la technologie '{1}' .",
   "tribes.hoodrick": "Hoodrick",
-  "tribes.hoodrick.info": "Les feuilles d'automne jaunes des bois des {0} sont des cachettes parfaites pour ces habitants farcis aux champignons. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.hoodrick.info": "Les feuilles d'automne jaunes des bois des {0} sont des cachettes parfaites pour ses habitants farcis aux champignons. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
```

Comment:
The both is correct...but it sounds better whit a demonstrative pronoun because in the firts part of the sentence the subject is at the passiv form.